### PR TITLE
chore(release): use cook and debugger only on dryRun

### PR DESCRIPTION
**Pull request by christianalfoni**

### CODE REVIEW

File: scripts/release/cooker.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ process.env.REPO_COOKER_GITHUB_TOKEN = process.env.GH_TOKEN
 
 export const cooker = Cooker({
   devtools:
-    !process.env.CI &&
+    dryRun &&
       new Devtools({
```
Comment (gaspard) [inline scripts/release/cooker.js]: Should be just the devtools settings: also no need to import Devtools.

```js
// This is what Cooker does:
const tools = new Devtools({
  host: options.devtools ? options.devtools.host : 'localhost:9090',
})
```

Comment (christianalfoni) [inline scripts/release/cooker.js]: oh, fixed! :-)

**Comment by Guria** [comment]
Seems debugger must be fixed in repo-cooker itself:
https://github.com/cerebral/repo-cooker/blob/259c57c725331b3f1c4057924851eac71f7d953d/src/Cooker/index.js#L22

**Comment by christianalfoni** [comment]
In what way @Guria ? Should only add devtools if devtools option passed in and not in production environment... which it does now?

**Comment by gaspard** [comment]
@christianalfoni Yes, I think we should only check for truthy value in repo-cooker. I'll add an issue with the other things to fix.

**Pull request merged**